test/schemas/view/FullName.schema

```diff
@@ -0,0 +1,4 @@
+{
+    FirstName: { domain: 'Nomen' },
+    LastName: { domain: 'Nomen' },
+}
```

Comment:
I think the indentation in schemas should be 2, isn't it?

Reply:
AFAIK it's not specified anywhere, we have no linting rules for schemas,  and indentation has no effect on it's parsing.